data_model/src/query.rs

```diff
@@ -7,27 +7,34 @@ use alloc::{boxed::Box, format, string::String, vec::Vec};
 
 use derive_more::Display;
 use iroha_crypto::SignatureOf;
+use iroha_data_model_derive::model;
 use iroha_macro::FromVariant;
 use iroha_schema::IntoSchema;
 use iroha_version::prelude::*;
 use parity_scale_codec::{Decode, Encode};
 use serde::{Deserialize, Serialize};
 
+pub use self::model::*;
 use self::{
     account::*, asset::*, block::*, domain::*, peer::*, permission::*, role::*, transaction::*,
     trigger::*,
 };
-use crate::{account::Account, model, Identifiable, Value};
+use crate::{account::Account, Identifiable, Value};
+
+macro_rules! queries {
+    ($($($meta:meta)* $item:item)+) => {
+        pub use self::model::*;
+
+        #[iroha_data_model_derive::model]
```

Comment:
If I'm not mistaken now you have `model` module inside `model` module? I would rather use `model_single`